pom.xml

```diff
@@ -716,8 +718,13 @@
     <!-- Profile for CI tests to have less output -->
     <profile>
       <id>ci-test</id>
+      <activation>
+        <property>
+          <name>ci-test-profile</name>
```

Comment:
Why enable it by default? BTW, do the surefire.xx arguments actually work locally?

Reply:
Generally, when you run `mvn test` you do not need the immense amount of low level logs provided now. It makes more sense for them to be opt-in. As for the other question, luckily these settings do work locally.